cogs/pycord.py

```diff
@@ -5,6 +5,9 @@
 
 from utils import Cog, pycord_only
 
+import re
+
+PASTEBIN_RE = re.compile(r"(http(?:s?)://pastebin.com)(?:/)(\w+?)(?!/\w+)\W")
```

Comment:
Suggestion by @IlluminatiFish 
```suggestion
PASTEBIN_RE = re.compile(r"(https?://pastebin.com)/([a-zA-Z0-9]{8})")
```